studio/pages/new/[slug].tsx

```diff
@@ -365,8 +368,8 @@ const FreeProjectLimitWarning = () => {
         description={
           <div className="space-y-3">
             <p className="text-sm leading-normal">
-              Your account can only have a maximum of 2 free projects. You can only choose paid
-              pricing plan.
+              {`Your account can only have a maximum of ${limit} free projects. You can only choose paid
+              pricing plan.`}
```

Comment:
could we reword this to:
`Your account can only have up to ${limit} free projects - to create another free project, you'll need to delete an existing free project first.`